### ISSUE TITLE
C9075093 Bug Fix - Lumberjack Tree Sprite Error

### ISSUE THREAD

**Issue by Dessyreqt** [Dessyreqt/alttprandomizer#280]
Submitter: Tantalus-X
have found i little funy texture bug ^^

http://www.directupload.net/file/d/4589/ckfypjup_png.htm

 Link to the Past Randomizer Log
---------------------------------
Version: 7
Creation Date: 03.01.2017 01:51:04
Complexity: 6
Seed: C9075093

Complexity Analysis
-------------------
Step 1 - Items reachable from start:
  [cave-021] Kakariko well [bottom chest]...................................................Bottle (Green Potion)
  [cave-021] Kakariko well [center chest row of 3]..........................................Cape
  [cave-022-B1] Thief's hut [bottom right chest]............................................Bottle (Red Potion)
  [cave-044] Aginah's cave..................................................................Flippers
  [cave-050] cave southwest of Lake Hylia [top right chest].................................Hookshot
  [dungeon-C-1F] Sanctuary..................................................................Magic Powder
  [dungeon-C-B1] Hyrule Castle - map room...................................................Moon Pearl
  [dungeon-L1-1F] Eastern Palace - Big key..................................................Bottle
  Piece of Heart (Dam)......................................................................Blue Mail
  Piece of Heart (Thieves' Forest Hideout)..................................................Bow

Step 2 - Items reachable after collecting all items from Step 1:
  [cave-063] doorless hut...................................................................Power Glove
  [cave-073] cave northeast of swamp palace - generous guy..................................Bug-Catching Net
  [cave-073] cave northeast of swamp palace [bottom chest]..................................Shovel
  [cave-073] cave northeast of swamp palace [top chest].....................................Fire Rod
  [dungeon-D3-B1] Skull Woods - big chest...................................................Red Mail
  [dungeon-D4-1F] Thieves' Town - Room above boss...........................................Book of Mudora
  [dungeon-D4-B1] Thieves' Town - Bottom left of huge room [bottom right chest].............Staff of Byrna
  [dungeon-D4-B1] Thieves' Town - Top left of huge room.....................................Mushroom
  Crystal - Thieves' Town...................................................................Crystal 1 (Orange/Red)
  Pendant - Eastern Palace..................................................................Pendant of Power (Blue)
  Piece of Heart (Dark World - bumper cave).................................................Pegasus Boots
  Piece of Heart (Treasure Chest Game)......................................................Titan's Mitt
  Piece of Heart (Zora's River).............................................................Ether
  Sick Kid..................................................................................Magic Mirror

Step 3 - Items reachable after collecting all items from Step 2:
  [cave-016] cave under rocks west of Santuary..............................................Boomerang
  [cave-057-1F] Dark World Death Mountain - cave from top to bottom [bottom chest]..........Quake
  [dungeon-L2-B1] Desert Palace - Map room..................................................Hammer
  Crystal - Skull Woods.....................................................................Crystal 5 (Green/Red)
  Ether.....................................................................................Red Shield
  Heart Container (Mothula).................................................................Bottle
  Magic Bat.................................................................................Cane of Somaria
  Pendant - Desert Palace...................................................................Pendant of Courage (Green)
  Pendant - Tower of Hera...................................................................Pendant of Wisdom (Red)
  Piece of Heart (Desert - west side).......................................................Lamp

Step 4 - Items reachable after collecting all items from Step 3:
  [dungeon-D2-B2] Swamp Palace - flooded room [right chest].................................Bombos
  [dungeon-D2-B2] Swamp Palace - hidden waterfall door room.................................Mirror Shield
  Crystal - Dark Palace.....................................................................Crystal 7 (Zelda)
  Crystal - Ice Palace......................................................................Crystal 4 (Pink/Red)
  Crystal - Swamp Palace....................................................................Crystal 3 (Yellow/Brown)
  Heart Container (Kholdstare)..............................................................Ocarina

Step 5 - Items reachable after collecting all items from Step 4:
  [dungeon-D7-1F] Turtle Rock - Map room [left chest].......................................Ice Rod
  Crystal - Misery Mire.....................................................................Crystal 6 (Green)

Step 6 - Items reachable after collecting all items from Step 5:
  [dungeon-A2-1F] Ganon's Tower - north of gap room [bottom right chest]....................Red Boomerang
  Crystal - Turtle Rock.....................................................................Crystal 2 (Pink/White)

Generated Item Order
--------------------
[cave-044] Aginah's cave..................................................................Flippers
[cave-021] Kakariko well [center chest row of 3]..........................................Cape
Piece of Heart (Thieves' Forest Hideout)..................................................Bow
[dungeon-L1-1F] Eastern Palace - Big key..................................................Bottle
[dungeon-C-B1] Hyrule Castle - map room...................................................Moon Pearl
[cave-050] cave southwest of Lake Hylia [top right chest].................................Hookshot
[dungeon-D4-1F] Thieves' Town - Room above boss...........................................Book of Mudora
Piece of Heart (Dark World - bumper cave).................................................Pegasus Boots
[cave-073] cave northeast of swamp palace [bottom chest]..................................Shovel
[cave-063] doorless hut...................................................................Power Glove
[dungeon-D4-B1] Thieves' Town - Top left of huge room.....................................Mushroom
[cave-073] cave northeast of swamp palace [top chest].....................................Fire Rod
[dungeon-L2-B1] Desert Palace - Map room..................................................Hammer
Piece of Heart (Treasure Chest Game)......................................................Titan's Mitt
Heart Container (Kholdstare)..............................................................Ocarina
Sick Kid..................................................................................Magic Mirror
[cave-057-1F] Dark World Death Mountain - cave from top to bottom [bottom chest]..........Quake
[dungeon-C-1F] Sanctuary..................................................................Magic Powder
[dungeon-D2-B2] Swamp Palace - flooded room [right chest].................................Bombos
Magic Bat.................................................................................Cane of Somaria

Light World
-----------
[cave-009-1F] Death Mountain - wall of caves - right cave [bottom chest]..................100 Rupees
[cave-009-1F] Death Mountain - wall of caves - right cave [top left chest]................3 Bombs
[cave-009-1F] Death Mountain - wall of caves - right cave [top left middle chest].........20 Rupees
[cave-009-1F] Death Mountain - wall of caves - right cave [top right chest]...............50 Rupees
[cave-009-1F] Death Mountain - wall of caves - right cave [top right middle chest]........3 Bombs
[cave-009-B1] Death Mountain - wall of caves - right cave [left chest]....................Heart Container
[cave-009-B1] Death Mountain - wall of caves - right cave [right chest]...................20 Rupees
[cave-012-1F] Death Mountain - wall of caves - left cave..................................20 Rupees
[cave-013] Mimic cave (from Turtle Rock)..................................................10 Arrows
[cave-016] cave under rocks west of Santuary..............................................Boomerang
[cave-018] Graveyard - top right grave....................................................Heart Container
[cave-021] Kakariko well [bottom chest]...................................................Bottle (Green Potion)
[cave-021] Kakariko well [center chest row of 3]..........................................Cape
[cave-021] Kakariko well [left chest row of 3]............................................1 Rupee
[cave-021] Kakariko well [right chest row of 3]...........................................50 Rupees
[cave-021] Kakariko well [top chest]......................................................Heart Container
[cave-022-B1] Thief's hut [bottom left chest].............................................Piece of Heart
[cave-022-B1] Thief's hut [bottom right chest]............................................Bottle (Red Potion)
[cave-022-B1] Thief's hut [top chest].....................................................10 Arrows
[cave-022-B1] Thief's hut [top left chest]................................................10 Arrows
[cave-022-B1] Thief's hut [top right chest]...............................................10 Arrows
[cave-026] chicken house..................................................................Heart Container
[cave-031] Tavern.........................................................................20 Rupees
[cave-034] Hyrule Castle secret entrance..................................................1 Rupee
[cave-035] Sahasrahla's Hut [center chest]................................................3 Bombs
[cave-035] Sahasrahla's Hut [left chest]..................................................3 Bombs
[cave-035] Sahasrahla's Hut [right chest].................................................20 Rupees
[cave-040] Link's House...................................................................100 Rupees
[cave-044] Aginah's cave..................................................................Flippers
[cave-047] Dam............................................................................20 Rupees
[cave-050] cave southwest of Lake Hylia - generous guy....................................10 Arrows
[cave-050] cave southwest of Lake Hylia [bottom left chest]...............................Piece of Heart
[cave-050] cave southwest of Lake Hylia [bottom right chest]..............................Piece of Heart
[cave-050] cave southwest of Lake Hylia [top left chest]..................................Piece of Heart
[cave-050] cave southwest of Lake Hylia [top right chest].................................Hookshot
[cave-051] Ice Cave.......................................................................Heart Container + Refill
Bombos....................................................................................20 Rupees
Bottle Vendor.............................................................................70 Arrow Cap
Ether.....................................................................................Red Shield
Haunted Grove item........................................................................Heart Container
Hobo......................................................................................20 Rupees
King Zora.................................................................................300 Rupees
Library...................................................................................50 Rupees
Mushroom..................................................................................20 Rupees
Old mountain man..........................................................................20 Rupees
Piece of Heart (Dam)......................................................................Blue Mail
Piece of Heart (Death Mountain - floating island).........................................50 Rupees
Piece of Heart (Desert - northeast corner)................................................Piece of Heart
Piece of Heart (Desert - west side).......................................................Lamp
Piece of Heart (Graveyard)................................................................100 Rupees
Piece of Heart (Lake Hylia)...............................................................20 Rupees
Piece of Heart (Lumberjack Tree)..........................................................20 Rupees
Piece of Heart (Maze Race)................................................................Piece of Heart
Piece of Heart (south of Haunted Grove)...................................................20 Rupees
Piece of Heart (Spectacle Rock Cave)......................................................3 Bombs
Piece of Heart (Spectacle Rock)...........................................................20 Rupees
Piece of Heart (Thieves' Forest Hideout)..................................................Bow
Piece of Heart (Zora's River).............................................................Ether
Purple Chest..............................................................................Piece of Heart
Sahasrahla................................................................................Piece of Heart
Sick Kid..................................................................................Magic Mirror
Witch.....................................................................................3 Bombs

Dark World
----------
[cave-055] Spike cave.....................................................................20 Rupees
[cave-056] Dark World Death Mountain - cave under boulder [bottom left chest].............20 Rupees
[cave-056] Dark World Death Mountain - cave under boulder [bottom right chest]............50 Rupees
[cave-056] Dark World Death Mountain - cave under boulder [top left chest]................10 Arrows
[cave-056] Dark World Death Mountain - cave under boulder [top right chest]...............3 Bombs
[cave-057-1F] Dark World Death Mountain - cave from top to bottom [bottom chest]..........Quake
[cave-057-1F] Dark World Death Mountain - cave from top to bottom [top chest].............20 Rupees
[cave-062] C-shaped house.................................................................20 Rupees
[cave-063] doorless hut...................................................................Power Glove
[cave-071] Misery Mire west area [left chest].............................................Piece of Heart
[cave-071] Misery Mire west area [right chest]............................................Heart Container
[cave-073] cave northeast of swamp palace - generous guy..................................Bug-Catching Net
[cave-073] cave northeast of swamp palace [bottom chest]..................................Shovel
[cave-073] cave northeast of swamp palace [bottom middle chest]...........................Heart Container
[cave-073] cave northeast of swamp palace [top chest].....................................Fire Rod
[cave-073] cave northeast of swamp palace [top middle chest]..............................Piece of Heart
Catfish...................................................................................Piece of Heart
Flute Boy.................................................................................300 Rupees
Piece of Heart (Dark World - bumper cave).................................................Pegasus Boots
Piece of Heart (Dark World blacksmith pegs)...............................................3 Bombs
Piece of Heart (Digging Game).............................................................20 Rupees
Piece of Heart (Pyramid)..................................................................Piece of Heart
Piece of Heart (Treasure Chest Game)......................................................Titan's Mitt

Hyrule Castle Escape
--------------------
[dungeon-C-1F] Sanctuary..................................................................Magic Powder
[dungeon-C-B1] Escape - final basement room [left chest]..................................100 Rupees
[dungeon-C-B1] Escape - final basement room [middle chest]................................Piece of Heart
[dungeon-C-B1] Escape - final basement room [right chest].................................3 Bombs
[dungeon-C-B1] Escape - first B1 room.....................................................Key
[dungeon-C-B1] Hyrule Castle - boomerang room.............................................Map
[dungeon-C-B1] Hyrule Castle - map room...................................................Moon Pearl
[dungeon-C-B3] Hyrule Castle - next to Zelda..............................................Piece of Heart

Eastern Palace
--------------
[dungeon-L1-1F] Eastern Palace - big ball room............................................Map
[dungeon-L1-1F] Eastern Palace - big chest................................................Compass
[dungeon-L1-1F] Eastern Palace - Big key..................................................Bottle
[dungeon-L1-1F] Eastern Palace - compass room.............................................10 Arrows
[dungeon-L1-1F] Eastern Palace - map room.................................................Big Key
Heart Container (Armos Knights)...........................................................300 Rupees

Desert Palace
-------------
[dungeon-L2-B1] Desert Palace - big chest.................................................Compass
[dungeon-L2-B1] Desert Palace - Big key room..............................................Big Key
[dungeon-L2-B1] Desert Palace - compass room..............................................Map
[dungeon-L2-B1] Desert Palace - Map room..................................................Hammer
Heart Container (Lanmolas)................................................................20 Rupees

Tower of Hera
-------------
[dungeon-L3-1F] Tower of Hera - first floor...............................................Big Key
[dungeon-L3-2F] Tower of Hera - Entrance..................................................Map
[dungeon-L3-4F] Tower of Hera - 4F [small chest]..........................................Compass
[dungeon-L3-4F] Tower of Hera - big chest.................................................Piece of Heart
Heart Container (Moldorm).................................................................Piece of Heart

Hyrule Castle Tower
-------------------
[dungeon-A1-2F] Hyrule Castle Tower - 2 knife guys room...................................Key
[dungeon-A1-3F] Hyrule Castle Tower - maze room...........................................Key

Dark Palace
-----------
[dungeon-D1-1F] Dark Palace - big chest...................................................Heart Container
[dungeon-D1-1F] Dark Palace - big key room................................................Key
[dungeon-D1-1F] Dark Palace - compass room................................................Piece of Heart
[dungeon-D1-1F] Dark Palace - jump room [left chest]......................................Key
[dungeon-D1-1F] Dark Palace - jump room [right chest].....................................Key
[dungeon-D1-1F] Dark Palace - maze room [bottom chest]....................................Map
[dungeon-D1-1F] Dark Palace - maze room [top chest].......................................Compass
[dungeon-D1-1F] Dark Palace - spike statue room...........................................3 Bombs
[dungeon-D1-1F] Dark Palace - statue push room............................................Key
[dungeon-D1-B1] Dark Palace - room leading to Helmasaur [left chest]......................Big Key
[dungeon-D1-B1] Dark Palace - room leading to Helmasaur [right chest].....................20 Rupees
[dungeon-D1-B1] Dark Palace - shooter room................................................Key
[dungeon-D1-B1] Dark Palace - turtle stalfos room.........................................Key
Heart Container (Helmasaur King)..........................................................10 Arrows

Swamp Palace
------------
[dungeon-D2-1F] Swamp Palace - first room.................................................Key
[dungeon-D2-B1] Swamp Palace - big chest..................................................20 Rupees
[dungeon-D2-B1] Swamp Palace - big key room...............................................50 Bomb Cap
[dungeon-D2-B1] Swamp Palace - map room...................................................Compass
[dungeon-D2-B1] Swamp Palace - push 4 blocks room.........................................Map
[dungeon-D2-B1] Swamp Palace - south of hookshot room.....................................3 Bombs
[dungeon-D2-B2] Swamp Palace - flooded room [left chest]..................................Big Key
[dungeon-D2-B2] Swamp Palace - flooded room [right chest].................................Bombos
[dungeon-D2-B2] Swamp Palace - hidden waterfall door room.................................Mirror Shield
Heart Container (Arrghus).................................................................20 Rupees

Skull Woods
-----------
[dungeon-D3-B1] Skull Woods - big chest...................................................Red Mail
[dungeon-D3-B1] Skull Woods - Big Key room................................................Compass
[dungeon-D3-B1] Skull Woods - Compass room................................................Big Key
[dungeon-D3-B1] Skull Woods - east of Fire Rod room.......................................Key
[dungeon-D3-B1] Skull Woods - Entrance to part 2..........................................Key
[dungeon-D3-B1] Skull Woods - Gibdo/Stalfos room..........................................Map
[dungeon-D3-B1] Skull Woods - south of Fire Rod room......................................Key
Heart Container (Mothula).................................................................Bottle

Thieves Town
------------
[dungeon-D4-1F] Thieves' Town - Room above boss...........................................Book of Mudora
[dungeon-D4-B1] Thieves' Town - Bottom left of huge room [bottom right chest].............Staff of Byrna
[dungeon-D4-B1] Thieves' Town - Bottom left of huge room [top left chest].................Key
[dungeon-D4-B1] Thieves' Town - Bottom right of huge room.................................Big Key
[dungeon-D4-B1] Thieves' Town - Top left of huge room.....................................Mushroom
[dungeon-D4-B2] Thieves' Town - big chest.................................................10 Arrows
[dungeon-D4-B2] Thieves' Town - next to Blind.............................................Map
Heart Container (Blind)...................................................................Compass

Ice Palace
----------
[dungeon-D5-B1] Ice Palace - Big Key room.................................................Key
[dungeon-D5-B1] Ice Palace - compass room.................................................Big Key
[dungeon-D5-B2] Ice Palace - map room.....................................................Compass
[dungeon-D5-B3] Ice Palace - spike room...................................................Key
[dungeon-D5-B4] Ice Palace - above Blue Mail room.........................................1 Arrow
[dungeon-D5-B5] Ice Palace - b5 up staircase..............................................Map
[dungeon-D5-B5] Ice Palace - big chest....................................................3 Bombs
Heart Container (Kholdstare)..............................................................Ocarina

Misery Mire
-----------
[dungeon-D6-B1] Misery Mire - big chest...................................................Piece of Heart
[dungeon-D6-B1] Misery Mire - big hub room................................................Big Key
[dungeon-D6-B1] Misery Mire - big key.....................................................Key
[dungeon-D6-B1] Misery Mire - compass.....................................................Compass
[dungeon-D6-B1] Misery Mire - end of bridge...............................................Key
[dungeon-D6-B1] Misery Mire - map room....................................................Piece of Heart
[dungeon-D6-B1] Misery Mire - spike room..................................................Key
Heart Container (Vitreous)................................................................Map

Turtle Rock
-----------
[dungeon-D7-1F] Turtle Rock - Chain chomp room............................................Big Key
[dungeon-D7-1F] Turtle Rock - compass room................................................Key
[dungeon-D7-1F] Turtle Rock - Map room [left chest].......................................Ice Rod
[dungeon-D7-1F] Turtle Rock - Map room [right chest]......................................Key
[dungeon-D7-B1] Turtle Rock - big chest...................................................Piece of Heart
[dungeon-D7-B1] Turtle Rock - big key room................................................Map
[dungeon-D7-B1] Turtle Rock - Roller switch room..........................................Key
[dungeon-D7-B2] Turtle Rock - Eye bridge room [bottom left chest].........................300 Rupees
[dungeon-D7-B2] Turtle Rock - Eye bridge room [bottom right chest]........................10 Arrows
[dungeon-D7-B2] Turtle Rock - Eye bridge room [top left chest]............................Key
[dungeon-D7-B2] Turtle Rock - Eye bridge room [top right chest]...........................100 Rupees
Heart Container (Trinexx).................................................................Compass

Ganon's Tower
-------------
[dungeon-A2-1F] Ganon's Tower - above Armos...............................................5 Rupees
[dungeon-A2-1F] Ganon's Tower - big chest.................................................Compass
[dungeon-A2-1F] Ganon's Tower - compass room [bottom left chest]..........................20 Rupees
[dungeon-A2-1F] Ganon's Tower - compass room [bottom right chest].........................Big Key
[dungeon-A2-1F] Ganon's Tower - compass room [top left chest].............................Key
[dungeon-A2-1F] Ganon's Tower - compass room [top right chest]............................Map
[dungeon-A2-1F] Ganon's Tower - down right staircase from entrance [left chest]...........3 Bombs
[dungeon-A2-1F] Ganon's Tower - down right staircase from entrance [right chest]..........Key
[dungeon-A2-1F] Ganon's Tower - east of down right staircase from entrace.................100 Rupees
[dungeon-A2-1F] Ganon's Tower - map room..................................................Piece of Heart
[dungeon-A2-1F] Ganon's Tower - north of gap room [bottom left chest].....................50 Rupees
[dungeon-A2-1F] Ganon's Tower - north of gap room [bottom right chest]....................Red Boomerang
[dungeon-A2-1F] Ganon's Tower - north of gap room [top left chest]........................Heart Container
[dungeon-A2-1F] Ganon's Tower - north of gap room [top right chest].......................1/4 Magic
[dungeon-A2-1F] Ganon's Tower - north of teleport room....................................Heart Container
[dungeon-A2-1F] Ganon's Tower - west of teleport room [bottom left chest].................Piece of Heart
[dungeon-A2-1F] Ganon's Tower - west of teleport room [bottom right chest]................3 Bombs
[dungeon-A2-1F] Ganon's Tower - west of teleport room [top left chest]....................3 Bombs
[dungeon-A2-1F] Ganon's Tower - west of teleport room [top right chest]...................Key
[dungeon-A2-6F] Ganon's Tower - before Moldorm............................................50 Rupees
[dungeon-A2-6F] Ganon's Tower - Moldorm room..............................................3 Bombs
[dungeon-A2-6F] Ganon's Tower - north of falling floor four torches [top left chest]......10 Arrows
[dungeon-A2-6F] Ganon's Tower - north of falling floor four torches [top right chest].....5 Rupees
[dungeon-A2-B1] Ganon's Tower - north of Armos room [bottom chest]........................Piece of Heart
[dungeon-A2-B1] Ganon's Tower - north of Armos room [left chest]..........................Piece of Heart
[dungeon-A2-B1] Ganon's Tower - north of Armos room [right chest].........................20 Rupees

Special Locations
-----------------
Bottle Item - Pyramid.....................................................................Bottle (Red Potion)
Bottle Item - Waterfall...................................................................Bottle (Red Potion)
Crystal - Dark Palace.....................................................................Crystal 7 (Zelda)
Crystal - Ice Palace......................................................................Crystal 4 (Pink/Red)
Crystal - Misery Mire.....................................................................Crystal 6 (Green)
Crystal - Skull Woods.....................................................................Crystal 5 (Green/Red)
Crystal - Swamp Palace....................................................................Crystal 3 (Yellow/Brown)
Crystal - Thieves' Town...................................................................Crystal 1 (Orange/Red)
Crystal - Turtle Rock.....................................................................Crystal 2 (Pink/White)
Pendant - Desert Palace...................................................................Pendant of Courage (Green)
Pendant - Eastern Palace..................................................................Pendant of Power (Blue)
Pendant - Tower of Hera...................................................................Pendant of Wisdom (Red)
Required Medallion - Misery Mire..........................................................Quake
Required Medallion - Turtle Rock..........................................................Bombos
Sword - Fairy.............................................................................Gold Sword
Sword - Pedestal..........................................................................Tempered Sword
Sword - Smithy............................................................................Master Sword

**Comment by BlueViper85** [comment]
This is a known issue, but the first bug report I've seen open on this. Investigation is on-going. Nothing was touched with regard to these specific sprites so the current theory is that this bug exists in Vanilla JP 1.0 but is showing its face now possibly as an unintended side effect of a Randomizer improvement or as a result of certain routing changes.

Thank you for the report. If you notice anything else like this or happen to have any thoughts on things you may have done in this route that may be different from usual please add them as a comment here.

**Comment by KatDevsGames** [comment]
How did this go this long without getting put on the github?

**Comment by BlueViper85** [comment]
I have a ROM and a video of a playthrough where this occurred.  I'm going to review the playthough and see what happens if I'm able to recreate their steps exactly hopefully in multiple emulators depending on the amount of time it will take.

I believe the source was running on a console via SD2SNES or similar, but if memory serves me well enough it happens on emulator too, signifying that this isn't just an issue with SD2SNES (or whatever was used)

Don't know if I'll get through it tomorrow or not, so it may be a few days before I can report back here

**Comment by thatguycore** [comment]
I know that I can tell you that the bug is not seed related from what I can tell. The other day I did a race where both I and the person I raced generated the same seed ourselves. We both ran on SNES9x 1.53 and he got the tree sprite bug and I did not.

**Comment by Radagast81** [comment]
yeah i'm allmost certain that the bug exists also in the vanilla version of the game as well. Could be related to the crystals/pedants you have collected, cause that is the one thing that changed the most until v6. And beforehand it wasn't this likely to show that bug...

**Comment by thatguycore** [comment]
[https://clips.twitch.tv/weffjebster/GleamingYakPuppeyFace](https://clips.twitch.tv/weffjebster/GleamingYakPuppeyFace)
[https://www.twitch.tv/weffjebster/v/111467489](https://www.twitch.tv/weffjebster/v/111467489)

This is the clip of the glitch and the run where it happened. The person is going to send me his copy of the generated rom when he gets off work. This glitch however happened on seed C20170101 (it is unwinnable).
Other seeds the glitch has appeared on as reported by people I chat with are C9161685 and C2788339.

I actually have the C9161685 rom that was generated. It was given to me in the middle of me typing this.

[ALttP_Random_-_C9161685.zip](https://github.com/Dessyreqt/alttprandomizer/files/683280/ALttP_Random_-_C9161685.zip)

**Comment by BlueViper85** [comment]
Following the steps of the above VoD in the same seed (C20170101(, I was able to recreate the Lumberjack Issue.

The general route was this:

-Get Sword and Shield
-Rescue Zelda
-Go through the town
-Activate the Flute
-Save the old man on the mountain
-Beat Eastern Palace
-Check what was possible in Hera
-Defeat agahnim and enter the DW as Bunny Link
-Get the hammer from the pyramid
-Mirror
-Go to the Lumberjack Tree

I have save states I a few places of this process.

A few other details:

-There were Game Over deaths in Agahnim's tower where we saved and quit.  Reloading a state before that showed the deaths had no part in it.
-Reloaded the state before Agahnim's tower and went through with no deaths and it still happened
-Confirmed it happened whether you got the hammer (and entering super bunny state) or not
-Reloaded the save state and Saved and Quit, started back at Link's house and went back through and killed agahnim immediately mirrored and went to the three and it was broken.
-Going to check the tree immediately after becoming bunny link on the mountain and then mirroring back gave me a normal tree as well.

So my own guess is that this is either A. because of going through all the major game state changing NPCs (Uncle, Save Zelda, Save the old man) in one sitting or possibly had to do with becoming Bunny Link after killing Agahnim, then going to the tree without saving and qutting 

I have more situations I want to test later as well.

**Comment by SmashManiac** [comment]
Reproduced the issue on v7 Casual seed C6433259. I had a really complex route so I can't really describe it, but what I can say is:
- My playthrough was in one sitting, although I died and save & quit very frequently.
- I saved the old man.
- I visited all dungeons in the overworld multiple times, and was only able to complete the Eastern Temple, which gave the Pendant of Wisdom.
- I fully upgraded my bombs/arrows at the nerfed fairy fountain, and later got the 70 arrows upgrade.
- I beat Agahnim without the Moon Pearl and noticed the issue about 5 minutes later.
- The issue fixed itself only after a save & quit. Entering a building was insufficient.

Considering this, I believe the Agahnim without Moon Pearl theory to be worth investigating.

I did not record my playthrough, but I did take a screenshot of when the issue happened:
![bug](https://cloud.githubusercontent.com/assets/22013850/21735951/4b279a5a-d43b-11e6-811e-ada8212d4127.png)
